modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/insect.dm

```diff
@@ -33,8 +33,8 @@
 	attack_effect = ATTACK_EFFECT_CLAW
 	attack_sound = 'sound/weapons/slash.ogg'
 	miss_sound = 'sound/weapons/slashmiss.ogg'
-	liked_food = GROSS | MEAT | TOXIC
-	disliked_food = JUNKFOOD | FRIED | CLOTH
+	liked_food = GROSS | RAW | TOXIC | CLOTH
```

Comment:
Try to keep it to 3/3 at max, normally races have 2 liked 3 disliked. Were pushing limits even at 3/3